platform/src/main/resources/config/anchor-config-default-values.yaml

```diff
@@ -716,6 +671,45 @@ assets:
   # The value of the assets definition depending on the `type` field.
   value:
 
+##############################
+# Anchor Platform Clients Configuration
+##############################
+# The list of clients for the Anchor server to safely communicate with the outside wallet servers or clients.
+# The list is empty if not defined.
+# You can either use a file to provide the configuration or define it inline.
+# Use only one of the following combinations:
+clients:
+  # Option 1: Use a file to provide the configuration.
+  # Specify the type as 'file' and provide the file path in the value field.
+  # The file can only be in .yaml or .json format.
+  # The file content should follow the format used in Option 2.
+  #
+  # type: file
+  # value: path/to/file.yaml
```

Comment:
Any reason not to remove `type` and `value` and replace it with `path`? Since `file` is the only valid `type`?

Reply:
This would be a breaking change I guess, so maybe we make it in 3.0?

Reply:
> Any reason not to remove `type` and `value` and replace it with `path`? Since `file` is the only valid `type`?
I thought about it, but taking a look at the whole config, `type` and `value` are always used in pair to reference files(e.g. sep1.toml, assets). I simply prefer to keep it consistent


> This would be a breaking change I guess, so maybe we make it in 3.0?
Yes this is for 3.0